mevislab.github.io/content/tutorials/basicmechanisms.md

```diff
@@ -4,63 +4,60 @@ date: 2022-06-15T08:54:53+02:00
 
-As a next step, we like to select and load the data we like to process. Double-click the Module `ImageLoad` to open its Panel. Now, you can browse through your folders to select the data you like to open. You can find example data in the MeVisLab DemoData directory *$(InstallDir)/Packages/MeVisLab/Resources/DemoData* located in the MeVisLab installation path. Select a file, for example an MRI scan of a shoulder *Shoulder_Fracture.tif*. The image is loaded immediately and basic information of the loaded image can be seen in the Panel.
+Next, we select and load the data we´d like to process. Double-click the Module `ImageLoad` to open its Panel. You can browse through your folders to select the data you´d like to open. Example data can be found in the MeVisLab DemoData directory *$(InstallDir)/Packages/MeVisLab/Resources/DemoData* located in the MeVisLab installation path. Select a file, for example an MRI scan of a shoulder *Shoulder_Fracture.tif*. The image is loaded immediately and basic information of the loaded image can be seen in the Panel.
 
 {{<alert class="info" caption="Extra Infos">}}
-Multiple additional Modules for loading data are available such as:
-* `DicomImport` for loading DICOM Images
-* `LocalImage` for loading any image format
+There also are modules to load multiple other formats of data. These are the most common ones:
+* `DicomImport` to load DICOM Images
+* `LocalImage` to load any image format
  
-For details about loading DICOM images, see {{< docuLinks "/Resources/Documentation/Publish/SDK/GettingStarted/ch03.html" "here">}}
+For a more detailed description on loading DICOM images, look {{< docuLinks "/Resources/Documentation/Publish/SDK/GettingStarted/ch03.html" "here">}}
 
 {{</alert>}}
 
 [//]: <> (MVL-651)
 
 ### The Output-Inspector and the Module Inspector {#The_Output_Inspector_and_the_Module_Inspector}
 
-For a first inspection and visualization of the loaded data, we can use the Output Inspector located in the {{< docuLinks "/Resources/Documentation/Publish/SDK/MeVisLabManual/ch04s09.html" "Views" >}} area. You can already interact with the image via Mouse-Wheel {{< mousebutton "middle" >}} and Mouse Buttons {{< mousebutton "left" >}} / {{< mousebutton "right" >}}. To preview the image, click on the triangle on the top side of the module `ImageLoad`, which offers the module output. All module outputs can be found at the top side of the respective module. 
+To inspect and visualize the loaded data, we can use the Output Inspector located in the {{< docuLinks "/Resources/Documentation/Publish/SDK/MeVisLabManual/ch04s09.html" "Views" >}} area. You can already interact with the image using the Mouse wheel {{< mousebutton "middle" >}} and Mouse Buttons {{< mousebutton "left" >}} / {{< mousebutton "right" >}}. To preview the image, click on the triangle on the top side of the Module `ImageLoad`, which offers the modules output. All Module outputs can be found at the top side of the respective Module. 
```

Comment:
Because I edited and rewrote some parts - but not every part. So I kept a lot of the initial paragraphs. 
You may recall that I wrote "mouse-button" and "mouse-wheel", I removed the hyphens once you told me that they are not supposed to be there, but I did not capitalize the words. At least I don't remember doing so. Will correct it nevertheless.